packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -66,7 +72,7 @@ export class ForceFunctionCreateExecutor extends BaseTemplateCommand {
             );
             reject(err);
           }
-          resolve();
+          resolve(void 0);
```

Comment:
Let's not make this change for consistency.

Reply:
Sounds good - I was seeing errors in VS Code for this, and it seems like there are some places in the code where we call it this way, but I'm happy to put it back.